datadogclient/formatter.go

```diff
@@ -33,6 +34,10 @@ func (f Formatter) Format(prefix string, maxPostBytes uint32, data map[metrics.M
 func (f Formatter) formatMetrics(prefix string, data map[metrics.MetricKey]metrics.MetricValue) []byte {
 	s := []metrics.Series{}
 	for key, mVal := range data {
+		// dogate feature
+		if strings.HasPrefix(key.Name, "bosh.healthmonitor") {
```

Comment:
I think you misread the feature:

```ruby
entry['metric'].sub!('datadog.nozzle.bosh-hm-forwarder', 'bosh.healthmonitor')
```

Reply:
Actually no, this is combined with https://github.com/DataDog/datadog-firehose-nozzle/pull/48/files#diff-cd6782160e51ec84de9b84e784c25799R47, for the full feature
one part replaces the metric name, and the other prevent appending the prefix `datadog.nozzle`

Reply:
ahh okay

Reply:
thanks!